apify-api/openapi/paths/actors/acts@{actorId}@runs@last.yaml

```diff
@@ -3,117 +3,65 @@ get:
     - Actors/Last run object and its storages
   summary: Get last run
   description: |
-    This is not a single endpoint, but an entire group of endpoints that lets
-    you to
-
-    retrieve and manage the last run of given Actor or any of its default
-    storages.
-
+    This is not a single endpoint, but an entire group of endpoints that lets you to
+    retrieve and manage the last run of given Actor or any of its default storages.
     All the endpoints require an authentication token.
 
-
     The endpoints accept the same HTTP methods and query parameters as
-
     the respective storage endpoints.
-
     The base path represents the last Actor run object is:
 
-
     `/v2/acts/{actorId}/runs/last{?token,status}`
 
-
-    Using the `status` query parameter you can ensure to only get a run with a
-    certain status
-
-    (e.g. `status=SUCCEEDED`). The output of this endpoint and other query
-    parameters
-
+    Using the `status` query parameter you can ensure to only get a run with a certain status
+    (e.g. `status=SUCCEEDED`). The output of this endpoint and other query parameters
     are the same as in the [Run object](#/reference/actors/run-object) endpoint.
 
-
-    In order to access the default storages of the last Actor run, i.e. log,
-    key-value store, dataset and request queue,
-
+    In order to access the default storages of the last Actor run, i.e. log, key-value store, dataset and request queue,
     use the following endpoints:
 
-
     * `/v2/acts/{actorId}/runs/last/log{?token,status}`
 
-
     * `/v2/acts/{actorId}/runs/last/key-value-store{?token,status}`
 
-
     * `/v2/acts/{actorId}/runs/last/dataset{?token,status}`
 
-
     * `/v2/acts/{actorId}/runs/last/request-queue{?token,status}`
 
-
     These API endpoints have the same usage as the equivalent storage endpoints.
-
     For example,
-
-    `/v2/acts/{actorId}/runs/last/key-value-store` has the same HTTP method and
-    parameters as the
-
+    `/v2/acts/{actorId}/runs/last/key-value-store` has the same HTTP method and parameters as the
     [Key-value store object](#/reference/key-value-stores/store-object) endpoint.
 
-
     Additionally, each of the above API endpoints supports all sub-endpoints
-
     of the original one:
 
-
     #### Key-value store
 
+    * `/v2/acts/{actorId}/runs/last/key-value-store/keys{?token,status}` [Key collection](#/reference/key-value-stores/key-collection)
 
```

Comment:
we dont need this blank line either, same with the other lists

not sure it it is rendered as two separate lists with it?

Reply:
Well, you were right. The Markdown parser in your head is quite admirable. 🫡

Addressed, and now it looks like this:

![image](https://github.com/user-attachments/assets/6413a7dc-be53-4283-bd49-397c7b290aa4)